rpcd/playbooks/roles/rpc_maas/templates/client.my.cnf.j2

```diff
@@ -2,5 +2,5 @@
 
 [client]
 host={{ container_address }}
-user={{ galera_root_user }}
-password={{ galera_root_password }}
+user={{ galera_root_user|default('root') }}
+password={{ galera_root_password|default('secrete') }}
```

Comment:
Hi @cloudnull, we're not setting any password though -- if for whatever reason galera_root_password doesn't exist in the environment (a problem in itself) then all that will happen is we will end up with an unusable .my.cnf.